README.md

```diff
@@ -184,7 +204,7 @@ Copy and paste the following vhost configuration inside that file:
 	<VirtualHost 127.0.0.1:80>
 	    ServerAdmin youremail@heysparkbox.com
 	    DocumentRoot "<your_path_to_files>/<your_project_name>/web"
-	    ServerName <your_project_name>.dev
+	    ServerName <your_project_name>.local
```

Comment:
Can we add a note about this change? https://laravel-news.com/chrome-63-now-forces-dev-domains-https 👍

Reply:
(I can do this in a subsequent PR/update too)